docs/output.md

```diff
@@ -194,6 +194,7 @@ Optionally, ASV sequences can be filtered for rRNA sequences identified by Barrn
 - `barrnap/`
   - `rrna.<kingdom>.gff`: GFF3 output for rRNA matches per kingdom, where kingdom is one of `bac,arc,mito,euk`.
   - `summary.tsv`: Summary of evalues for each ASV and kingdom
+  - `result.tsv`: Classification of ASVs into kindoms based on the lowest evalue, only if `--filter_ssu` is set.
```

Comment:
```suggestion
  - `result.tsv`: Classification of ASVs into domains based on the lowest evalue, only if `--filter_ssu` is set.
```
I was going to correct `kindoms` to `kingdoms` but barrnap is actually domains, i.e. bacteria, archaea etc.

Reply:
Ah thanks, indeed!